analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Eval Static Analysis"
+author: "Chakshu Goyal"
+date: "07/06/2018"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(tools)
+```
+
+## Load the data
+Usage of the function eval in R-packages
```

Comment:
It is always good to put a command and other info about how to actually recreate the evals.csv